website/docs/components/table/table/partials/guidelines/guidelines.md

```diff
@@ -159,45 +152,44 @@ When using striping in a Table, the first tinted row will be the second row of t
+In order for the user to sort and read the table easily, each cell in a table should contain one piece of data. Having more than one piece of information inside a cell makes it harder for users to navigate the relationships between headers and content. 
 
 ### Density
 
-- We recommend using `medium` cell density by default.
+- Use `medium` density by default for balanced readability and display.
+- Choose `short` density for text-heavy tables to fit more rows on a page but can make tables harder to read and scan, so use it thoughtfully.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
-- If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
 
 ![Table cell density](/assets/components/table/table-density.png)
 
 ## Horizontal scrolling
-Use horizontal scrolling when columns in a Table need to expand beyond the standard Table container. We recommend only implementing horizontal scrolling when data becomes difficult to read in a single view. Use the [Advanced Table](components/table/advanced-table), when keyboard navigation and sticky columns ease the reading experience for large data-sets.
+Use horizontal scrolling when columns need to expand beyond the standard container. We recommend only implementing horizontal scrolling when data becomes difficult to read in a single view. Use the [Advanced Table](components/table/advanced-table), when keyboard navigation and sticky columns ease the reading experience for large data-sets.
 
 For more information on the types of approaches used to implement horizontal scrolling, refer to the [code tab](/components/table/table?tab=code#scrollable-table)
 
 ## Multi-select
 
-A multi-select table includes checkboxes enabling users to select multiple rows in a table to perform bulk operations. Checking or unchecking the checkbox in the table header either selects or deselects the checkboxes on each row in the table body. Individual checkboxes in the rows can also be selected or deselected.
+Multi-select allows users to select multiple rows to perform bulk actions, such as deleting or exporting data. 
 
 A multi-select consists of: 
 
-1. A select all in a table is used in the table's header row. This acts as the parent checkbox, allowing the selection or deselection of all child rows in a single table simultaneously.
+1. A select all in the table's header row to allow the selection or deselection of all child rows simultaneously.
```

Comment:
[suggestion] Since these have been turned into full sentences, I would remove line 186 and the numbering from this and line 192.

```suggestion
A "Select all" checkbox is used in the header row to allow the simultaneous selection or deselection of all child rows.
```